Knock.net/Resources/Objects/ObjectsResource.cs

```diff
@@ -396,8 +396,8 @@ public async Task<PreferenceSet> SetWorkflowPreferences(string collection, strin
         /// </summary>
         /// <param name="collection">Collection name.</param>
         /// <param name="objectId">Object unique identifier.</param>
-        /// <param name="options">Options for pagination</param>
-        /// <returns>A paginated list of ObjectSunscription records.</returns>
+        /// <param name="options">Options for filtering and pagination.</param>
+        /// <returns>A paginated list of ObjectSubscription records.</returns>
         public async Task<PaginatedResponse<ObjectSubscription>> ListSubscriptions(string collection, string objectId, Dictionary<string, object> options = null)
```

Comment:
looks like the string, object type can have an object of any type in this case it's a list so unsure if any update is necessary